src/components/Article.jsx

```diff
@@ -62,17 +64,37 @@ if(loading){
             <p id="createdAt"> Created at: {getArticle.created_at}</p>
             </div>
             <h3 id="commentHeader">Comments</h3>
-            <section id="comments">
-                <div id="addCommentsWrapper">
-                    <div id="authorContainer">
-                <img id="authorAvatar" src={props.getUser.avatar_url}></img>
-                 <p id="author"> Username: {props.getUser.username}</p>
+            <section id="addComments">
+                    <div id="UserContainer">
+                        {
+                            props.getUser.username ? <img id="userAvatar" src={props.getUser.avatar_url}></img> : null
+                        }
+                        {
+                            props.getUser.username ?  <p id="username2"> Username: {props.getUser.username}</p> : <p id="noUsername"> Please choose a user to be able to make a comment</p>
+                        }
+               
                     </div>
-                </div>
-                <li id="comments list">
-
-                </li>
+                <label id="commentLable" htmlFor="commentArea">Comment Below</label>
+                <textarea id="commentArea"></textarea>
+               <button className="commentSubmitButton">Submit Comment</button>
             </section>
+                <div id="comments">
+                       { 
+                            commentsList.map(comment => {
+                                return <ul className="comment">
+                                    <label className="commentLables" htmlFor="commentId">Comment ID</label>
+                                    <p className="commentText">{comment.comment_id} </p>
+                                    <label className="commentLables" htmlFor="commentAuthor">Author</label>
+                                    <p className="commentText">{comment.author}</p>
+                                    <label className="commentLables" htmlFor="commentBody">Comment</label>
+                                    <p className="commentText">{comment.body}</p>
+                                    <label className="commentLables" htmlFor="createdAt">Date Created</label>
+                                    <p id="createdAt">{comment.created_at}</p>
+                                    <hr id="line"></hr>
+                                </ul>
+                            })
+                       }
+                </div>
```

Comment:
It looks like you've gotten into a bit of a muddle over HTML tags here. I'll explain what I mean:

- `<ul>` is an **unordered list** and is a container for lots of individual items in a list, each of which should be represented by an `<li>` - **list item**. Currently you are rendering a `<div>` that contains lots of comments, each represented by its own `<ul>`, which is semantically confusing. I recommend looking at the HTML of your app in the devtools and having a read of [the docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) to compare what you should hope to see when rendering a list! 
- `<label>` is a tag which represents a caption for something that a user interacts with, whereas here they are just associated with static data being rendered on the page. If you want to label content like this you would be better served to use a `<p>` tag or `<h3/4/5>`

On that second point.... maybe not everything needs a label? https://www.refactoringui.com/previews/labels-are-a-last-resort (**NOTE:** this change is not mandatory but it's a nice read to get us thinking about good UX!)

Reply:
I have changed the ul to a div and changed the labels to p tag. Hopefully this sorts this?